### PR TITLE
Go Language Syntax Support

### PR DESCRIPTION
**Pull request by LoveMHz**
Added initial support for Go language syntax.

### CODE REVIEW

**Comment by appleboy** [comment]
@LoveMHz Could you resolve the code conflicts?

**Comment by LoveMHz** [comment]
What code conflicts?

**Comment by Viper007Bond** [comment]
I don't know why the tests are being dumb and saying there's a conflict. I'll manually merge and push it out.

**Comment by appleboy** [comment]
@Viper007Bond Nice work!

File: syntaxhighlighter.php [hunk 1]
```diff
@@ -184,6 +185,7 @@ function __construct() {
 			'erl'           => 'erlang',
 			'erlang'        => 'erlang',
 			'fsharp'        => 'fsharp',
+			'go'            => 'go',
```
Comment (appleboy) [inline syntaxhighlighter.php:193]: Please add another syntax `golang`.

**Comment by LoveMHz** [comment]
Added golang. Sorry for the delay!

**Comment by appleboy** [comment]
@Viper007Bond Any updates?

File: syntaxhighlighter.php (hunk 1 above)
Comment (LoveMHz) [inline syntaxhighlighter.php:193]: Done

Comment (appleboy) [inline syntaxhighlighter.php:193]: 👍 💯

**Comment by Viper007Bond** [comment]
Nope, been busy, sorry. In the middle of trying to move apartments among other things.

**Comment by Viper007Bond** [comment]
If you're super impatient and can't wait (sorry!), it's easy to add additional languages to my plugin without modifying it: http://www.viper007bond.com/wordpress-plugins/syntaxhighlighter/adding-a-new-brush-language/

**Comment by donnchawp** [comment]
@LoveMHz Still showing conflicts, but I've copied to the Automattic fork of the repo. https://github.com/Automattic/syntaxhighlighter/pull/5

**Comment by federicoleon** [comment]
Hi there!! Any news on this PR?? Do we have an ETA for merging this? @Viper007Bond @LoveMHZ ?

**Comment by pacuna** [comment]
Please merge this!

**Comment by alexsanford** [comment]
@LoveMHz could you please rebase this onto the latest `master`, and ensure it is only making the required changes? I see, for example, several changes to `readme.txt` and other brush files which do not appear to be related to this PR.

If we can get that cleaned up, then we should be able to get this merged 🙂

**Comment by diegoacuna** [comment]
@LoveMHz nice work on this, any chances on having this merged any time soon? if you don't have the time maybe we can open a new PR with the required changes on the code

**Comment by smarquezs** [comment]
Please merge this!

**Comment by alexsanford** [comment]
@diegoacuna please feel free to open a new PR if you would like. This one needs some work, and it seems the most recent interaction from the original author was in 2016. If we can get a PR up and working that only makes the required changes, then we should be able to get it merged 🙂

**Comment by yscik** [comment]
Done in #194

**Pull request closed without merging**